### DISCUSSION TITLE
Limiting multiple user logins on one computer

### DISCUSSION THREAD

**Discussion by vlaex** [Glench/ExtPay#158] [Ideas]
Hi!

Currently, ExtensionPay allows users to create multiple accounts using different email addresses on the same computer.
Example: you start the trial with `mail1@gmail.com`, then the trial is expiring, you reinstall the extension and you can start the trial with another email, like `mail2@gmail.com`.
This can lead to potential misuse of the system by users starting new trials repeatedly with different emails.

A solution that restricts the number of connected accounts (email addresses) per one computer would be very useful. This means that once a user logs in with a specific email address on a computer, there should be no way to log in with another email address on the same computer, even after reinstalling the extension.

**Comment by surtic86** [comment]
II had this Problem on my Extension to. I would suggest to implement  something like https://github.com/disposable/disposable to check for Disposable E-Mails.

**Reply by Glench** [reply]
We already detect and block disposable email address services.

**Reply by surtic86** [reply]
Oh so you have upgraded this in the last months?

Good!

**Reply by burkybang** [reply]
Yes, I remember seeing it announced in the newsletter or somewhere else.

**Comment by Glench** [comment]
Unfortunately there is no way to reliably detect "logins from one computer". IP address would be the closest thing, but those can change and there would be many support issues with users on the same network if IP addresses were used to block trials.

At some point we'll add the ability for trials to optionally require cards. That should be coming in the next big update.
